client/app/resources/collections-api.factory.js

```diff
@@ -58,6 +58,9 @@
       options = options || {};
 
       if (options.expand) {
+        if (angular.isArray(options.expand)) {
+          options.expand = options.expand.join(',');
+        }
```

Comment:
well, it's not strictly necessary in that we could just pass `expand: 'resources,service_requests'`, but it's to make it more consistent with `attributes` and `decorators`, which already accept arrays..